factcast-store/src/test/resources/test_schema.sql

```diff
@@ -1,123 +1,227 @@
-#
-CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
-
-DROP TRIGGER IF EXISTS tr_fact_insert ON fact;
-DROP SEQUENCE IF EXISTS catchup_seq;
-DROP TABLE IF EXISTS fact CASCADE;
-DROP TABLE IF EXISTS catchup CASCADE;
-DROP TABLE IF EXISTS schemastore cascade;
-DROP TABLE IF EXISTS tokenstore cascade;
-DROP TABLE IF EXISTS snapshot_cache cascade;
-DROP TABLE IF EXISTS transformationstore cascade;
-DROP TABLE IF EXISTS transformationcache cascade;
-
-CREATE TABLE fact (
- ser SERIAL PRIMARY KEY,
- 
- header JSONB NOT NULL,
- payload JSONB NOT NULL
-
-);
-
-#
-
-CREATE OR REPLACE FUNCTION notifyFactInsert() RETURNS trigger AS $$
-BEGIN
-  PERFORM pg_notify('fact_insert', json_build_object(
-    'ser', NEW.ser,
-    'header',NEW.header
-  )::text);
-  RETURN NEW;
+# CREATE
+  EXTENSION IF NOT EXISTS "uuid-ossp";
+
+DROP
+  TRIGGER IF EXISTS tr_fact_insert ON
+  fact;
+
+DROP
+  SEQUENCE IF EXISTS catchup_seq;
+
+DROP
+  TABLE
+    IF EXISTS fact CASCADE;
+
+DROP
+  TABLE
+    IF EXISTS catchup CASCADE;
+
+DROP
+  TABLE
+    IF EXISTS schemastore CASCADE;
+
+DROP
+  TABLE
+    IF EXISTS tokenstore CASCADE;
+
+DROP
+  TABLE
+    IF EXISTS snapshot_cache CASCADE;
+
+DROP
+  TABLE
+    IF EXISTS transformationstore CASCADE;
+
+DROP
+  TABLE
+    IF EXISTS transformationcache CASCADE;
+
+CREATE
+  TABLE
+    fact(
+      ser SERIAL PRIMARY KEY,
+      header JSONB NOT NULL,
+      payload JSONB NOT NULL
+    );
+
+# CREATE
+  OR REPLACE FUNCTION notifyFactInsert() RETURNS TRIGGER AS $$ BEGIN PERFORM pg_notify(
```

Comment:
Still wondering what the # sign does here. Comments in SQL would be "--"...

Apply formatter again after fixing this and check what happens...

```suggestion
CREATE OR REPLACE FUNCTION notifyFactInsert() RETURNS TRIGGER AS $$ BEGIN PERFORM pg_notify(
```